js/capture.js

```diff
@@ -10,10 +10,25 @@ const capture = {
   addListeners() {
     // listen for each HTTP response
     browser.webRequest.onResponseStarted.addListener(
-      this.sendThirdParty,
+      (response) => this.sendThirdParty(response),
```

Comment:
Are you saying to update it like this?:
```javascript  
 addListeners() {
    // listen for each HTTP response
    browser.webRequest.onResponseStarted.addListener(
      (response) => {
        this.sendThirdParty(response);
      },
      {urls: ['<all_urls>']});
    // ...
```
Which method returns a promise?

You are concerned that without the curly braces, the argument may not be read correctly by the webRequest API as a callback function? Could you explain a little more.